Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -580,7 +580,7 @@ bool AccessibilityNodeObject::computeAccessibilityIsIgnored() const
             return true;
 
         // Whitespace only text elements should be ignored when they have no renderer.
-        String string = stringValue().stripWhiteSpace().simplifyWhiteSpace();
+        String string = stringValue().stripLeadingAndTrailingCharacters(deprecatedIsSpaceOrNewline).simplifyWhiteSpace();
```

Comment:
Since this call site is just checking if all characters are “spaces”:

1. There is no need to call `simplifyWhiteSpace` since that never turns a non-empty string into an empty string.
2. There is no need to use the `String` function that allocates memory instead of the `StringView` function that is more efficient since it does not.
3. It could use a function that returns a `bool` instead of a `String`. I think we have such functions on `StringView` names containsOnly perhaps.

Reply:
Nice! Done. Also filed https://bugs.webkit.org/show_bug.cgi?id=257251 as follow-up as `containsOnly` seems like a better name.